CONTRIBUTING.md

```diff
@@ -378,33 +378,33 @@ Unsure where to begin contributing to Atom? You can start by looking through the
 
 ### Yarn
 
-See the full list of commands [here](https://classic.yarnpkg.com/en/docs/cli/).  
+See the full list of commands [here](https://classic.yarnpkg.com/en/docs/cli/).
 
-`yarn install`  
-Installs all node modules  
-Similar to `npm i`  
+`yarn install`
+Installs all node modules
+Similar to `npm i`
```

Comment:
These changes shouldn't be here as it adds unnecessary file changes to the PR

Reply:
I haven't touches this file, not sure why it even showed up in changes, but should be fixed in the last commit